src/Chainweb/Pact/SPV.hs

```diff
@@ -114,7 +114,8 @@ verifySPV cid cdbv typ proof = readMVar cdbv >>= go typ proof
             --  3. Extract tx outputs as a pact object and return the
             --  object.
 
-            TransactionOutput p <- verifyTransactionOutputProof cdb u
+            bh <- runTransactionOutputProof u
+            TransactionOutput p <- verifyTransactionOutputProofAt cdb u bh
```

Comment:
This code is trivially true. It confirms that the target of the proof is an ancestor of the target of the proof. What needs to be confirmed is that the target of the proof is an ancestor of the currently validated block.